layouts/partials/header.html

```diff
@@ -13,6 +13,10 @@
           <div class="nav-item-wrapper">
             <a href="{{ "index.xml" | relURL }}" class="nav-item-content" target="_blank">RSS</a>
           </div>
+          <div id="dark-mode-container">
+            <i id="dark-mode-icon" class="fas fa-sun"></i>
```

Comment:
i think we do not include `fontawesome` yet. does the `fas fa-sun`work well?

Reply:
they do work well and i use the same on my site.